study1/week4/ValidPalindrome2.java

```diff
@@ -0,0 +1,38 @@
+class Solution {
+    public boolean validPalindrome(String s) {
+        int end = s.length() - 1;
+
+        for(int i = 0; i < s.length()/2 ; i++){
```

Comment:
naming `i` to start should make better sense as you named the last character `end`